spk/wallabag/src/installer.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+# Package
+PACKAGE="wallabag"
+DNAME="Wallabag"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+WEB_DIR="/var/services/web"
+USER="$([ $(grep buildnumber /etc.defaults/VERSION | cut -d"\"" -f2) -ge 4418 ] && echo -n http || echo -n nobody)"
+TMP_DIR="${SYNOPKG_PKGDEST}/../../@tmp"
+
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Install the web interface
+    cp -R ${INSTALL_DIR}/share/${PACKAGE} ${WEB_DIR}
+
+    # permissions
+    chown -R ${USER} ${WEB_DIR}/${PACKAGE}
+
+    exit 0
+}
+
+preuninst ()
+{
+    exit 0
+}
+
+postuninst ()
+{
+    # Remove link
+    rm -f ${INSTALL_DIR}
+
+    # Remove the web interface
+    rm -fr ${WEB_DIR}/${PACKAGE}
+
+    exit 0
+}
+
+preupgrade ()
+{	
+    rm -fr ${TMP_DIR}/${PACKAGE}
+    mkdir -p ${TMP_DIR}/${PACKAGE}
+    mv ${WEB_DIR}/${PACKAGE}/inc/poche/config.inc.php ${TMP_DIR}/${PACKAGE}/
+    mv ${WEB_DIR}/${PACKAGE}/db ${TMP_DIR}/${PACKAGE}/
+    exit 0
+}
+
+postupgrade ()
+{
+    rm -fr ${WEB_DIR}/${PACKAGE}/cache/*
```

Comment:
I'm reasonably sure these two entries (this and next line) are simply unnecessary. After an upgrade, there should not be any content in `cache` and `install`, so clearing it won't do anything.